From 9224d1f8b594ae4e08c51708cd7484ec6c0a8594 Mon Sep 17 00:00:00 2001
From: sezero <sezero@users.sourceforge.net>
Date: Tue, 20 Mar 2018 15:01:02 +0300
Subject: [PATCH] os h: fix misplaced _V_IFDEFJAIL_H_ #else

I don't see the need for _V_IFDEFJAIL_H_ ifdef'ery, but libvorbis still
has it, so...
---
 os.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/os.h b/os.h
index 329c5d0..06e3612 100644
--- a/os.h
+++ b/os.h
@@ -35,9 +35,9 @@
 #    define STIN static __inline
 #  elif defined(__WATCOMC__)
 #    define STIN static __inline
+#  else
+#    define STIN static
 #  endif
-#else
-#  define STIN static
 #endif
 
 #ifdef _WIN32
-- 
1.7.1

